.github/workflows/ci.yaml

```diff
@@ -29,27 +29,6 @@ jobs:
       - name: Git Checkout
         uses: actions/checkout@v2
 
-      - name: Cache Ivy
```

Comment:
Caching is essential and should not be removed. It's a lot of traffic if we fetch all the dependencies every time we run a CI. That caching makes a performance hit is a nice find. Instead of removing it altogether, we should minimize its impact.

Reply:
The caching issues have been addressed by #10197.